### PR TITLE
Starting spec with a separate command

### PR DESCRIPTION
**Pull request by askonev**
1. The solution to the problem of not running a separate speck using the docker compose script.

  If you write the name of the spec in a variable, the command docker-compose up will start this spec. If we leave the variable unchanged, all specs will be started

[compose/environment-variables](https://docs.docker.com/compose/environment-variables/#the-env-file)

2. There is also a solution to running several commands from the docker-compose script. The solution presented in the documentation did not work.

  ```ruby
      command: ["./wait-for-it.sh", "db:5432", "--", "python", "app.py"]
  ```

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -0,0 +1,2 @@
+DOCSERVER_VERSION=latest
+SPEC_TYPE=
```
Comment (ShockwaveNN) [inline .env:2]: I think this option should have default value, like `DOCSERVER_VERSION` to understand which option can be placed here

File: docker-compose.yml [hunk 2]
```diff
@@ -22,6 +22,8 @@ services:
       - nginx
     environment:
       - DOCUMENTSERVER=http://documentserver
-    command: ["ruby", "./helpers/wait_for_documentserver_start.rb"]
+    command: >
+      bash -c "ruby helpers/wait_for_documentserver_start.rb
```
Comment (ShockwaveNN) [inline docker-compose.yml:26]: Maybe place both commands to entrypoint.sh like [this](https://github.com/ONLYOFFICE/testing-wrata/blob/master/entrypoint.sh)

This allow more clear calls in docker-compose

**Review by ShockwaveNN — CHANGES_REQUESTED** [review]

**Comment by askonev** [comment]
Missed the rebase again((((

File: Gemfile.lock [hunk 3]
```diff
@@ -5,7 +5,7 @@ GEM
       public_suffix (>= 2.0.2, < 5.0)
     ast (2.4.2)
     aws-eventstream (1.2.0)
-    aws-partitions (1.532.0)
+    aws-partitions (1.533.0)
```
Comment (ShockwaveNN) [inline Gemfile.lock:8]: Please rebase it around `master` to not include this changes

File: entrypoint.sh [hunk 4]
```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+ruby ./helpers/wait_for_documentserver_start.rb
+rspec
```
Comment (ShockwaveNN) [inline entrypoint.sh:4]: Do you removed ability to run custom `rspec` at all?

Do you plan to make it in seprarte PR (I think it's a great idea)

**Review by ShockwaveNN — CHANGES_REQUESTED** [review]

File: .env (hunk 1 above)
Comment (askonev) [reply .env]: The parameter is left blank because by default I want to run all the tests in the project with the `docker-compose up` command.

**Comment by askonev** [comment]
How about putting the CURRENT_SPEC in the docker file?

File: .env (hunk 1 above)
Comment (ShockwaveNN) [reply .env]: In that case maybe add one example and comment it out

**Comment by ShockwaveNN** [comment]
> How about putting the CURRENT_SPEC in the docker file?

No, I think it's better to have all parameters in one place - in your case idea with `.env` file looks good

**Review by ShockwaveNN — CHANGES_REQUESTED** [review]
.

**Review by ShockwaveNN — APPROVED** [review]

**Pull request merged**